modules/integration_azure-storage-file-premium/common-versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
```

Comment:
why not symbolic link for those kind of files?

Reply:
Wrong copy/paste :)